docs/modules/ROOT/pages/explanations/app_catalog.adoc

```diff
@@ -0,0 +1,118 @@
+= Application Catalog Concept
+
+TIP: For an introduction what the application catalog is, please see https://products.docs.vshn.ch/products/appcat/index.html[our product documentation].
+
+The application catalog consists of:
+
+* *Control-Plane* to deploy, configure, update and maintain services
+* *Service Catalog* which contains a list of well-defined services
+* *Framework* to build services upon
+
+It is our framework to fully automate service provisioning and all tasks for day-2 operations.
+
+image::appcat_concept.drawio.svg[]
+
+== Service Location
+
+Services can be provisioned on various places:
+
+Local Cluster::
```

Comment:
Maybe call it control cluster?

Reply:
I would distinguish between these names, as the control cluster doesn't necessarily need to run services.

Reply:
I agree that distinguishing them is a good thing. But local and remote are a bit unclear.

Maybe something like this:
control cluster -> cluster dedicated to hosting the controlplane
service cluster -> cluster dedicated to running the services
converged cluster -> cluster running both and other customer services if neccassry.

Should help with ambiguities I think.